addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -2163,7 +2163,14 @@ var SnippetsMenu = Widget.extend({
             this.invisibleDOMMap = new Map();
             const $invisibleDOMPanelEl = $(this.invisibleDOMPanelEl);
             $invisibleDOMPanelEl.find('.o_we_invisible_entry').remove();
-            const $invisibleSnippets = globalSelector.all().find('.o_snippet_invisible').addBack('.o_snippet_invisible');
+            let isMobile;
```

Comment:
You can squash both commits IMO. Like explained in my other comment, I don't think it make sense to have the feature without the possibility of not having to switch device display anyway.